virttest/utils_net.py

```diff
@@ -3386,26 +3386,41 @@ def get_ip_address_by_interface(ifname, ip_ver="ipv4", linklocal=False):
         return None
 
 
-def get_host_ip_address(params, ip_ver="ipv4", linklocal=False):
+def get_host_ip_address(params=None, ip_ver="ipv4", linklocal=False, ifname=None):
```

Comment:
I think no need to add extra parameter ifname, just specify it in param

Reply:
In tp-qemu, they need to check ip_addr from given host_ip_addr or netdst parameter. we have netdst either.
But I want to get ip_address from default or given interface name.
So I add a new param ifname here, and when it's None, it will get ip address from default interface.

Reply:
@kylazhang Updated.